lib/lbt/bundle/Builder.js

```diff
@@ -288,7 +288,7 @@ class BundleBuilder {
 		const sequence = section.modules.slice();
 
 		this.beforeWriteFunctionPreloadSection(sequence);
-
+		this.outW.writeln(`//@ui5-bundle ${section.bundle.name}`);
```

Comment:
also part of #341 . Not sure if the auto-merge can handle this.

Reply:
Rebase can :)